scripts/analytics/node_log_events.py

```diff
@@ -0,0 +1,92 @@
+import time
+import logging
+import argparse
+from modularodm import Q
+from datetime import datetime, timedelta
+from dateutil.parser import parse
+
+from website.app import init_app
+from website.project.model import NodeLog
+from website.settings import KEEN as keen_settings
+from keen.client import KeenClient
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def get_events(date):
+    """ Get all node logs from a given date for a 24 hour period.
+    Defaults to starting yesterday and gathering through today (both in UTC).
+    """
+
+    # Make sure default is yesterday and not today
+    if date.day == datetime.utcnow().day:
+        date = date - timedelta(1)
```

Comment:
Since this may end up getting called from different places, I wouldn't provide a default.  Instead raise an error if the caller doesn't pass in a date, or if the date is unsatisfactory (i.e. in the future or today when UTC is still < 23:59:59).  Force the caller to think about what they're asking for.